example/1-pipeline/Taskfile.yml

```diff
@@ -19,8 +19,7 @@ tasks:
   sfn:
     desc: sfn
     cmds:
-      # - "yomo run -n rand serverless/rand.go" # use latest yomo
-      - "yomo run -m go.mod -n rand serverless/rand.go" # use custom go.mod for development
+      - "yomo build serverless/rand.go && yomo run serverless/sfn.wasm -n rand"
```

Comment:
Custom go.mod builds should be used to avoid yomo version incompatibility。

Reply:
pushed commit below